packages/next/build/babel/preset.ts

```diff
@@ -116,7 +116,7 @@ module.exports = (
           ...options['preset-react'],
         },
       ],
-      require('@babel/preset-typescript'),
+      [require('@babel/preset-typescript'), { allowNamespaces: true }],
```

Comment:
may I have a question? why there no provide options by next.config? Is seem like next takes full control of @babel/preset-typescript.